src/objects/core/zcl_abapgit_file_status.clas.abap

```diff
@@ -762,17 +763,18 @@ CLASS ZCL_ABAPGIT_FILE_STATUS IMPLEMENTATION.
     DATA lt_remote TYPE zif_abapgit_definitions=>ty_files_tt.
     DATA li_exit TYPE REF TO zif_abapgit_exit.
 
-    lt_local = io_repo->get_files_local( ii_log ).
+    lt_local = io_repo->get_files_local( ii_log = ii_log
+                                         ii_pre_filter = ii_pre_filter ).
 
     IF lines( lt_local ) <= 2.
       " Less equal two means that we have only the .abapgit.xml and the package in
       " our local repository. In this case we have to update our local .abapgit.xml
       " from the remote one. Otherwise we get errors when e.g. the folder starting
       " folder is different.
-      io_repo->find_remote_dot_abapgit( ).
+      io_repo->find_remote_dot_abapgit( ii_pre_filter = ii_pre_filter ).
```

Comment:
this looks suspicious, the remote dot abapgit should not be possible to filter away